juju-terraform-provider-k8s-controller-full-bundle/terraform.tfvars.example

```diff
@@ -0,0 +1,9 @@
+# Provider configuration
+k8s_juju_controller_address = "10.246.167.235:17070"
+k8s_juju_controller_username = "admin"
+k8s_juju_controller_password  = "9caccd638e58ab0da11172d1ca37a519"
+k8s_juju_controller_ca_certificate_path = "data/juju-cert.pem"
+
+# Terraform Bundle configuration
+juju_cloud = "kubernetes_cloud"
+juju_model_name = "kubeflow"
```

Comment:
We can hardcode this for now, some of our charms expect the namespace/model to always be `kubeflow`

Reply:
Perfect, let me push a hardcoded version